.github/CODEOWNERS

```diff
@@ -480,3 +480,7 @@ Makefile                                              @thiagokokada
 /modules/services/plex-mpv-shim.nix                   @starcraft66
 
 /modules/services/sctd.nix                            @somasis
+
+/modules/programs/aerc.nix                            @lukasngl
+/modules/programs/aerc-accounts.nix                   @lukasngl
+/tests/modules/programs/aerc                          @lukasngl
```

Comment:
This file is alphabetized, so this should go before `/modules/programs/aria2.nix`